### ISSUE TITLE
CognitoUserPoolTriggerHandler is depreciated in newer version of aws-lambda

### ISSUE THREAD

**Issue by jamie-wearsafe** [aws-samples/amazon-cognito-passwordless-email-auth#30]
Looking for the correct type.

```sh
@deprecated — Please use specific event handler types instead

'CognitoUserPoolTriggerHandler' is deprecated ts(6385)
```

**Comment by martinpllu** [comment]
Hi @jamie-wearsafe, I hit this issue too, and after a bit of searching around was able to use the following:

```
import { CreateAuthChallengeTriggerHandler } from 'aws-lambda';
import { DefineAuthChallengeTriggerHandler } from 'aws-lambda';
import { PostAuthenticationTriggerHandler } from 'aws-lambda';
import { PreSignUpTriggerHandler } from 'aws-lambda';
import { VerifyAuthChallengeResponseTriggerHandler } from 'aws-lambda';
```

**Comment by ottokruse** [comment]
Thanks! Fixed in #31
